### PR TITLE
IMC metadata changes

### PR DESCRIPTION
**Pull request by cebriggs7135**
Several filed related to offsets have been removed. This information will be provided in metadata .csv files provided in the metadata directory.

### CODE REVIEW

File: src/ingest_validation_tools/table-schemas/level-2/imc.yaml [hunk 1]
```diff
@@ -64,127 +67,39 @@ fields:
...
-    entity is stored as binary
-  type: number
-- name: data_start_offset_units
-  description: The units of the offset value.
-  constraints:
-    enum:
-      - bytes
+- name: acquisition_id
+  description: The acquisition_id refers to the filename for a slide image. Together, the acquisition_id and the roi_id
+  indicate the slide-ROI represented in the image.
```
Comment (cebriggs7135) [inline src/ingest_validation_tools/table-schemas/level-2/imc.yaml:77]:
```suggestion
  description: The acquisition_id refers to the directory containing the ROI images for a slide. Together, the acquisition_id and the roi_id
  indicate the slide-ROI represented in the image.
```

File: src/ingest_validation_tools/table-schemas/level-2/imc.yaml [hunk 2]
```diff
@@ -40,18 +40,21 @@ fields:
 - name: reagent_prep_protocols_io_doi
   description: DOI for protocols.io referring to the protocol for preparing reagents
     for the assay.
+- name: number_of_channels
+  description: Number of antibodies imaged.
+  type: number
```
Comment (cebriggs7135) [inline src/ingest_validation_tools/table-schemas/level-2/imc.yaml:45]:
```suggestion
- name: number_of_channels
  description: Number of mass channels measured
  type: number
```

File: src/ingest_validation_tools/table-schemas/level-2/imc.yaml [hunk 3]
```diff
@@ -40,18 +40,21 @@ fields:
...
 - name: ablation_distance_between_shots_x_value
-  description: Distance (\xB5m) between laser ablation shots in the X-dimension.
+  description: x resolution. Distance (\xB5m) between laser ablation shots in the X-dimension.
```
Comment (mccalluc) [inline src/ingest_validation_tools/table-schemas/level-2/imc.yaml:47]:
```suggestion
  description: x resolution. Distance between laser ablation shots in the X-dimension.
```
unit is separate.

File: src/ingest_validation_tools/table-schemas/level-2/imc.yaml [hunk 4]
```diff
@@ -40,18 +40,21 @@ fields:
...
 - name: ablation_distance_between_shots_x_units
-  description: Units of distance between laser ablation shots.
+  description: Units of x resolution distance between laser ablation shots.
   constraints:
     enum:
       - um
       - nm
 - name: ablation_distance_between_shots_y_value
-  description: Distance (\xB5m) between laser ablation shots in the Y-dimension.
+  description: y resolution. Distance (\xB5m) between laser ablation shots in the Y-dimension.
```
Comment (mccalluc) [inline src/ingest_validation_tools/table-schemas/level-2/imc.yaml:55]:
```suggestion
  description: y resolution. Distance between laser ablation shots in the Y-dimension.
```

**Comment by mccalluc** [comment]
> This information will be provided in metadata .csv files provided in the metadata directory.

@cebriggs7135 : I'll go ahead and get this merged, but double check with Nils about that... His default is for metadata to go into this file, so there needs to be a pretty clear rationale if it is to be separate.

**Comment by cebriggs7135** [comment]
It is metadata that qualifies as level 3 or 4 and will be provided in a metadata file clearly identified as offset values for reading the entities that are stored in the MCD files as binaries. It is not relevant to the experiment or viewing or interpretation off the data.

**Pull request merged**